README.md

```diff
@@ -11,12 +11,24 @@ A tool like [google-photos-exif](https://github.com/mattwilson1024/google-photos
 
 ## Run this tool
 
+If you wish to migrate a single folder from a Google Photos takeout file:
+
+```bash
+mkdir output error
+
+npx google-photos-migrate@latest migrateFolder '/path/to/takeout/Google Fotos' './output' './error' --timeout 60000
+```
+
+If you wish to migrate an entire takeout folder:
+
 ```bash
 mkdir output error
 
-npx google-photos-migrate@latest '/path/to/takeout/Google Fotos' './output' './error' --timeout 60000
+npx google-photos-migrate@latest migrateFolder '/path/to/takeout/' --timeout 60000
```

Comment:
This appears to be the wrong command.  It's `fullMigrate`

Reply:
From the errors I got, and what the code says, the instructions (to match the current code) should actually be:

```
mkdir -p /path/to/takeout/{Albums,Photos}

npx google-photos-migrate@latest fullMigrate '/path/to/takeout' --timeout 60000
```